qmcpy/discrete_distribution/lattice/lattice.py

```diff
@@ -79,10 +107,11 @@ def __init__(self, dimension=1, randomize=True, order='natural', seed=None,
                 Note: Non-randomized lattice sequence includes the origin.
             order (str): 'linear', 'natural', or 'mps' ordering.
             seed (None or int or numpy.random.SeedSeq): seed the random number generator for reproducibility
-            generating_vector (ndarray or str): generating matrix or path to generating matrices.
+            generating_vector (ndarray, str or int): generating matrix or path to generating matrices.
                 ndarray should have shape (d_max).
                 a string generating_vector should be formatted like
                 'lattice_vec.3600.20.npy' where 'name.d_max.m_max.npy'
+                an integer should be larger than 1
```

Comment:
add more explanation here as to what an integer means. Maybe something like "passing an integer M will create a random generating vector supporting up to 2^M points. m is restricted to be between 2 and 26 for numerical precision. The randomly generated vector is [1,v_1,v_2,...] where v_i is an integer in [3,2^M-1]"